defaultConfig.stub.js

```diff
@@ -870,8 +870,8 @@ module.exports = {
 
   options: {
     prefix: '',
-    separator: '\\:',
     important: false,
+    separator: '\\:',
```

Comment:
Didn't you just change this to simply `:`?

Reply:
Yep but neither is merged yet :)